CHANGELOG.md

```diff
@@ -40,6 +40,10 @@ collected logs to standard output. This can be useful for debugging. (@jcreixell
   - `otelcol.receiver.jeager` receives Jaeger-formatted traces. Data can then
     be forwarded to other `otelcol` components. (@rfratto)
 
+- Introduce `/-/support` endpoint for generating 'support bundles'; zip files
+  of commonly-requested information that can be used to debug a running agent.
+  (@tpaschalis)
```

Comment:
Should we make this clear this is for the non-Flow ("static") agent?

Reply:
Yeah, if you're also ok with wiring this to Flow in a different PR. I'll update the changelog to reflect that.

Reply:
> Yeah, if you're also ok with wiring this to Flow in a different PR. I'll update the changelog to reflect that.

Yeah, it's ok to put Flow in another PR 👍 (though if we close out the original issue we'll want to create a new issue for Flow support)